web/src/main/kotlin/no/nav/su/se/bakover/web/routes/revurdering/LeggTilUtlandsoppholdRoute.kt

```diff
@@ -0,0 +1,65 @@
+package no.nav.su.se.bakover.web.routes.revurdering
+
+import arrow.core.getOrHandle
+import io.ktor.application.call
+import io.ktor.http.HttpStatusCode
+import io.ktor.routing.Route
+import io.ktor.routing.post
+import no.nav.su.se.bakover.common.serialize
+import no.nav.su.se.bakover.domain.Brukerrolle
+import no.nav.su.se.bakover.service.revurdering.KunneIkkeLeggeTilUtlandsopphold
+import no.nav.su.se.bakover.service.revurdering.RevurderingService
+import no.nav.su.se.bakover.service.vilkår.LeggTilOppholdIUtlandetRequest
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.errorJson
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.routes.Feilresponser
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBody
+import no.nav.su.se.bakover.web.withRevurderingId
+import java.util.UUID
+
+private data class UtlandsoppholdBody(
```

Comment:
Vi må vel ha et eller annet forhold til periode når det kommer til revurdering?

Reply:
Vi kan ta utgangspunkt i at perioden == revurderingsperioden dersom vi kun tar inn ett element. Men jeg mtp. hvor smertefullt det er med bosituasjon. Ville jeg vel og anbefale og kunne ha flere elementer for revurdering.

Reply:
Løses.